settings.gradle

```diff
@@ -6,8 +6,8 @@ rootProject.name = 'emp3-android'
 include 'google-maps-android-cmgn' // Google Maps android utils
 project(':google-maps-android-cmgn').projectDir = new File(settingsDir, '3rdpartylibrary/google-maps-android-cmgn')
 
-include 'worldwind-release' // NASA WorldWind Android
-project(':worldwind-release').projectDir = new File(settingsDir, '3rdpartylibrary/worldwind-release')
+//include 'worldwind-release' // NASA WorldWind Android
+//project(':worldwind-release').projectDir = new File(settingsDir, '3rdpartylibrary/worldwind-release')
```

Comment:
Overall, I am OK with you completely removing these statements and not having them continue to exist as commented-out code.